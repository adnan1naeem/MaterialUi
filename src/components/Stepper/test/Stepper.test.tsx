import { Stepper } from '../';
import renderer from 'react-test-renderer';
import React from 'react';
test('render correctly', () => {
	const Steppertest = renderer.create(
    <Stepper  type="progress" activeStep={3} steps={6}   />
	).toJSON();
	expect(Steppertest).toBeDefined();
	expect(Steppertest).toMatchSnapshot();
});