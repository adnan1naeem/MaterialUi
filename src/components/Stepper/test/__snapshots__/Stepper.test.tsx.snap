// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="MuiPaper-root-9 MuiPaper-shadow0-11 MuiMobileStepper-root-1 MuiMobileStepper-positionBottom-2"
>
  <div
    className="MuiMobileStepper-progress-8"
  >
    <div
      aria-valuenow={60}
      className="MuiLinearProgress-root-36 MuiLinearProgress-primaryColor-37"
    >
      <div
        className="MuiLinearProgress-bar-43 MuiLinearProgress-primaryColorBar-38 MuiLinearProgress-determinateBar1-50"
        style={
          Object {
            "transform": "scaleX(0.6)",
          }
        }
      />
    </div>
  </div>
</div>
`;
